.github/workflows/luacheck.yml

```diff
@@ -0,0 +1,16 @@
+name: Luacheck
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  luacheck:
+    runs-on: ubuntu-latest
```

Comment:
I always have mixed feelings on `-latest`. It's nice to run latest, but sometimes it gives you an annoying surprise. No specific opinion either way.

Reply:
I'll stick with `-latest` for now, and will likely reconsider if/when I get burned :)